config/locales/ar.yml

```
@@ -418,4 +418,4 @@ ar:
     series: "السلاسل العلمية"
     news: "الأخبار"
     about: "حول"
-    contact: "اتصل بنا"
+    contact: "اتصل بنا"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)